harness/harness.rb

```diff
@@ -1,51 +1,38 @@
-require 'csv'
+require 'benchmark'
 
 # Warmup iterations
-$WARMUP_ITRS = ENV.fetch('WARMUP_ITRS', 15).to_i
+WARMUP_ITRS = ENV.fetch('WARMUP_ITRS', 15).to_i
 
 # Minimum number of benchmarking iterations
-$MIN_BENCH_ITRS = 10
+MIN_BENCH_ITRS = ENV.fetch('MIN_BENCH_ITRS', 10).to_i
 
 # Minimum benchmarking time in seconds
-$MIN_BENCH_TIME = 10
+MIN_BENCH_TIME = ENV.fetch('MIN_BENCH_TIME', 10).to_i
 
-$out_csv_path = File.expand_path(ENV.fetch('OUT_CSV_PATH', 'output.csv'))
+OUT_CSV_PATH = ENV.fetch('OUT_CSV_PATH', "results-#{RUBY_ENGINE}-#{RUBY_ENGINE_VERSION}.csv")
```

Comment:
If you get rid of the File.expand_path, I think a benchmark that changes the current directory will stop working unless you manually supply an absolute path on the command line.

Reply:
Right, I'll restore the `File.expand_path`.